include/sdf/Filesystem.hh

```diff
@@ -128,6 +128,14 @@ namespace sdf
       IGN_UTILS_UNIQUE_IMPL_PTR(dataPtr)
     };
   }
+
+  /// \brief The source path replacement if it was parsed from a string,
+  /// instead of a file.
```

Comment:
nit Instead of `const std::string`, consider instead using `constexpr char[]` since you're in C++17 land.

Reply:
https://github.com/osrf/sdformat/pull/537/commits/745ea3bf520de5c16927d2dccade26bd3db73b85